dimod/core/bqm.py

```diff
@@ -499,6 +518,22 @@ def energies(self, samples_like, dtype=None):
         return np.asarray(energies, dtype=dtype)  # handle any type promotions
 
     def energy(self, sample, dtype=None):
+        """Determine the energy of the given sample.
+
+        Args:
+            samples_like (samples_like):
+                Raw sample. `samples_like` is an extension of
+                NumPy's array_like structure. See :func:`.as_samples`.
+
+            dtype (data-type, optional, default=None):
+                Desired NumPy data type for the energy. Matches
+                :attr:`.dtype` by default.
+
+        Returns:
+            :obj:`numpy.dtype`: The energy.
```

Comment:
It's really a numpy [scalar](https://numpy.org/doc/stable/reference/arrays.scalars.html), I would just say "The energy".